index.html

```diff
@@ -41,6 +41,7 @@ <h1>🌱 Script Seed</h1>
                     <option value="python">Python</option>
                     <option value="ruby">Ruby</option>
                     <option value="scala">Scala</option>
+                    <option value="perl">Perl</option>
```

Comment:
These lists are in alphabetical order. Not a big deal - I'll merge it and rearrange them.